src/backer/cardaning.py

```diff
@@ -143,18 +138,15 @@ def confirmTrans(self):
                     continue
 
                 item = json.loads(item)
-                (blockSlot, blockHeight) = (item["block_slot"], int(item["block_height"])) if 'block_slot' in item.keys() else (0, 0)
-                publishTime = datetime.fromisoformat(item["publish_time"])
-                onChainTime = help.toIso8601(self.slotToTime(blockSlot))
-                onChainTime = datetime.fromisoformat(onChainTime).replace(tzinfo=timezone.utc)
-                confirmTime = datetime.fromisoformat(help.toIso8601())
+                transTip = int(item["tip"])
+                blockHeight = int(item["block_height"]) if 'block_height' in item.keys() else 0
 
                 # Check for confirmation
                 if self.tipHeight > 0 and blockHeight > 0 and self.tipHeight - blockHeight >= TRANSACTION_SECURITY_DEPTH:
                     self.keldbConfirming.rem(keys)
                     continue
 
-                if (confirmTime - publishTime).total_seconds() > TRANSACTION_CONFIRMATION_TIMEOUT:
+                if self.tipHeight - transTip > TRANSACTION_TIMEOUT_DEPTH:
                     self.submitKelTx(item['kel'].encode('utf-8'))
```

Comment:
This should also be `publishEvent` right?

Reply:
Yes, This should be pushed back to pending KEL to make "events batch" :)